batsTests/startup.bats

```diff
@@ -0,0 +1,71 @@
+#! /bin/bash
+# Bind an unbound BATS variables that fail all tests when combined with 'set -o nounset'
+export BATS_TEST_START_TIME="0"
+export BATSLIB_FILE_PATH_REM=""
+export BATSLIB_FILE_PATH_ADD=""
+
+load '/workspace/target/bats_libs/bats-support/load.bash'
+load '/workspace/target/bats_libs/bats-assert/load.bash'
+load '/workspace/target/bats_libs/bats-mock/load.bash'
+load '/workspace/target/bats_libs/bats-file/load.bash'
+
+setup() {
+  export STARTUP_DIR=/workspace/resources
+  export WORKDIR=/workspace
+  netstat="$(mock_create)"
+  export netstat
+  export PATH="${BATS_TMPDIR}:${PATH}"
+  ln -s "${netstat}" "${BATS_TMPDIR}/netstat"
+}
+
+teardown() {
+  unset STARTUP_DIR
+  unset WORKDIR
+  rm "${BATS_TMPDIR}/netstat"
+}
+
+@test "create_hba() should use cidr 16 if the dogu is running in a k8s cluster" {
+  mock_set_output "${netstat}" "Kernel-IP-Routentabelle
+Ziel            Router          Genmask         Flags   MSS Fenster irtt Iface
+192.168.179.0   0.0.0.0         255.255.255.0   U         0 0          0 wlp0s20f3"
```

Comment:
shouldn't the genmask be 255.255.255.255 to test the behaviour?

Reply:
Recap of a side discussion: a genmask of ...0 is sufficient as test because it would still limit the available IP addresses to 256 which is still not enough for a cluster of several nodes. We keep it this way.